seed/src/lib.rs

```diff
@@ -222,28 +226,27 @@ impl Node {
     /// Handle user requests.
     async fn handle_request(request: Request, api: &Peer<Signer>) -> Result<(), Error> {
         match request {
-            Request::GetMembership(mut reply) => {
+            Request::GetMembership(reply) => {
                 let info = api.membership().await;
-                reply.send(info).await?;
+                reply.send(info).map_err(|_| Error::Reply)
```

Comment:
Does this need the closure?

Reply:
Ah, my bad, I misread the code and thought it was passing argument to `Error::Reply`, not enough coffee!,